cmd/witness/cmd/sign.go

```diff
@@ -44,11 +46,27 @@ func SignCmd() *cobra.Command {
 //todo: this logic should be broken out and moved to pkg/
 //we need to abstract where keys are coming from, etc
 func runSign(so options.SignOptions) error {
-	signer, err := loadSigner(so.KeyOptions.SpiffePath, so.KeyOptions.KeyPath, so.KeyOptions.CertPath, so.KeyOptions.IntermediatePaths)
-	if err != nil {
-		return err
+	ctx := context.Background()
+	signers, errors := loadSigners(ctx, so.KeyOptions)
+	if len(errors) > 0 {
+		for _, err := range errors {
+			log.Error(err)
+		}
+		return fmt.Errorf("failed to load signers")
+	}
+
+	if len(signers) > 1 {
+		log.Error("only one signer is supported")
+		return fmt.Errorf("only one signer is supported")
```

Comment:
same comment here as in the other function -- logging should be handled at the function that's calling this if possible.  similar to here, if we can: https://github.com/testifysec/witness/blob/e194af87ef8474f9f333b0075ed9c7bb83570c10/cmd/witness/cmd/root.go#L52